backend/src/main/java/edu/pl/masi/yellow/entity/UserEntity.java

```diff
@@ -0,0 +1,61 @@
+package edu.pl.masi.yellow.entity;
+
+import javax.persistence.*;
+
+@Entity
+@Table(name="user_data")
+public class UserEntity {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private int id;
+
+    @Column(name="user_name")
+    private String username;
+
+    @Column(name="user_pass")
+    private String password;
+
+    @Column(name="user_role")
+    private String role;
+
+    public UserEntity(String username, String password) {
+        this.username = username;
+        this.password = password;
+        this.role = "user";
+    }
+
+    public UserEntity() {
```

Comment:
Do we need empty constructor?

Reply:
Default constructors are mostly used by frameworks for dependency injection or JSON serialization